src/codeu/chat/server/Server.java

```diff
@@ -142,6 +147,22 @@ public void onMessage(InputStream in, OutputStream out) throws IOException {
         Serializers.collection(ConversationHeader.SERIALIZER).write(out, conversations);
       }
     });
+    
+//    // Get Server Info - **for Up Time function**
+//    //					A client wants to know the time the server was started
+//    //					updated by Julia 5/22
+//    
+    this.commands.put(NetworkCode.SERVER_INFO_REQUEST, new Command() {
+    	@Override
+        public void onMessage(InputStream in, OutputStream out) throws IOException {
+    		final ServerInfo info = new ServerInfo();
+    		
+          Serializers.INTEGER.write(out, NetworkCode.SERVER_INFO_RESPONSE);
```

Comment:
I honestly haven't looked at this, but is `info.startTime()` an `int`? Or is it maybe a `long` or something?

I'd try printing out the value here just for debugging. Is the date weird here too?